src/_P183_modbus.ino

```diff
@@ -234,8 +236,9 @@ boolean Plugin_183(uint8_t function, struct EventStruct *event, String& string)
                                    P183_GET_FLAG_COLL_DETECT)) {
         break;
       }
+      addLogMove(LOG_LEVEL_INFO, "P183 INIT AFTER INIT");
       P183_ModbusDevice->setModbusTimeout(P183_MODBUS_TIMEOUT);
-
+      addLogMove(LOG_LEVEL_DEBUG, "AFTER TIMEOUT");
       # ifdef P183_DEBUG
 
       if (loglevelActiveFor(LOG_LEVEL_DEBUG)) {
```

Comment:
You do check here for `P183_DEBUG`. Make sure this can only be enabled when `BUILD_NO_DEBUG` is not defined.